web/packages/teleterm/package.json

```diff
@@ -32,7 +32,7 @@
     "strip-ansi": "^7.1.0",
     "tar-fs": "^3.0.6",
     "which": "^5.0.0",
-    "winston": "^3.14.2"
+    "winston": "^3.17.0"
```

Comment:
`message` is `unknown[]` because in `createLoggerFromWinston` we collect all the arguments into an array:
https://github.com/gravitational/teleport/blob/381704f42ab9e15670e5f9e1ee1719f413154449/web/packages/teleterm/src/services/logger/loggerService.ts#L165-L177

We could add a runtime check instead of type assertion, but since we control the message, I'm leaning towards keeping it as is. We have the assertions in other places, I added it here as well.

Reply:
Yeah, I don't quite understand how they imagine people are going to use this new type. It's just a straight up `unknown` without any generics in play.